### DISCUSSION TITLE
Does it work with ESP32 DEVKITV1 or ESP32-CAM?

### DISCUSSION THREAD

**Discussion by ERedstone** [risinek/esp32-wifi-penetration-tool#63] [Q&A] [answered]
i am new to ESP32 and was wondering which one to buy for this project.

**Comment by IvarNilsson** [comment, marked as answer]
I personally use the devkits since they have more gpio pins out and are generally very simple to use. This gives them more flexibility for different projects. 

If you still want a cam and a lot of gpio-pins the "FREENOVE ESP32-WROVER CAM" is a great option but more slightly pricey.
